src/Capabilities_Command.php

```diff
@@ -66,38 +64,35 @@ class Capabilities_Command extends WP_CLI_Command {
 	 */
 	public function list_( $args, $assoc_args ) {
 		$role_obj = self::get_role( $args[0] );
-		
+
 		$show_grant = ! empty( $assoc_args['show-grant'] );
 
 		if ( $show_grant ) {
 			array_push( $this->fields, 'grant' );
 			$capabilities = $role_obj->capabilities;
-		} 
-		else {
+		} else {
 			$capabilities = array_filter( $role_obj->capabilities );
 		}
-		
+
 		$output_caps = array();
 		foreach ( $capabilities as $cap => $grant ) {
-			$output_cap = new StdClass;
+			$output_cap = new StdClass();
```

Comment:
The class name is camelCase here:
```suggestion
			$output_cap = new stdClass();
```